pylint_odoo/checkers/no_modules.py

```diff
@@ -371,11 +377,22 @@ def colon_list_to_dict(self, colon_list):
         """
         return dict([item.split(":") for item in colon_list])
 
+    def replace_and_first_lower(self, words):
+        """Convert 'Hello World' to 'hello_World'
```

Comment:
You have a method to convert field name to label name, right?
A capitalized name is a label one ("Hello World"), the correct field name is not "hello_World", the correct one is: "hello_world"
You will see my point when fix the tests like as my other comment